src/pages/index.js

```diff
@@ -55,13 +55,22 @@ const IndexPage = () => {
   }, [state.user])
 
   useEffect(() => {
-    if (form.lessonDescription !== 'DEFAULT') {
+    if (form.lessonDescription !== 'DEFAULT' && signUpComplete(state.user)) {
       setMeetingButton()
     }
   }, [form.lessonDescription])
 
+  useEffect(() => {
+    if (signUpComplete(state.user)) {
+      setForm({...form, isLoading: false})
+    }
+   
+  }, [form.showMeetingButton])
```

Comment:
Codacy found an issue: [React Hook useEffect has missing dependencies: 'form' and 'state.user'. Either include them or remove the dependency array. You can also do a functional update 'setForm(f => ...)' if you only need 'form' in the 'setForm' call.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5919912)